GitCommands/Patches/PatchFile.cs

```diff
@@ -19,19 +19,25 @@ public class PatchFile
         public bool IsNext { get; set; }
 
         public bool IsSkipped { get; set; }
+        public bool IsApplied { get; set; }
 
         public string Status
         {
             get
             {
-                if (IsNext)
+                if (IsSkipped)
                 {
-                    return "Next to apply";
+                    return "Skipped";
                 }
 
-                if (IsSkipped)
+                if (IsApplied)
                 {
-                    return "Skipped";
+                    return "Applied";
+                }
+
+                if (IsNext)
+                {
+                    return "Applying...";
```

Comment:
These should be translated 🤔

Reply:
Oh! Thanks a lot for the remainder.

I planned to do it but I didn't remember if I already looked at it and it was impossible due to #6402 or if I purely forgot to do it....

I will have a look!

Reply:
Unfortunately, that's due to #6402. We have to fix it before being able to translate it 😢 
I didn't remember but I also wrote it in the commit message: https://github.com/gitextensions/gitextensions/pull/6531/commits/ec4180ea5a40bc8b90374ac454dde369def995dd

Reply:
Thank you for checking.